src/tests/suite/daos_pool.c

```diff
@@ -610,9 +608,7 @@ pool_properties(void **state)
 		fail_msg("scrubber threshold verification failed.\n");
 	}
 
-	if (arg->myrank == 0)
-		daos_debug_set_params(arg->group, -1, DMG_KEY_FAIL_LOC, 0,
-				     0, NULL);
+	test_set_engine_fail_loc(arg, CRT_NO_RANK, 0);
```

Comment:
I guess there is no harm to specify CRT_NO_RANK here , though to match the earlier call to test_set_engine_fail_loc(), could the rank argument in this call be info.pi_leader? And same question would apply to pool_op_retry()

Reply:
Here I remember I was following what the test originally intended. That said, clearing the fail_loc of every engine is more robust as well simpler (e.g., "Hmm, what if the leader changes? Has there been any leader changes...seems unlikely...") in these tests. So I'd probably do the same. Does this sound remotely acceptable? :)

(I did have some concern that the "set param" mechanism uses CoRPCs, rather than an out-of-DAOS-band mechanism, so that it might not work independently of what a test has done. If there's any such issues preventing `CRT_NO_RANK` from working, we will need to revisit this.)